scripts/gh-review-threads.sh

```
@@ -0,0 +1,443 @@
+    done
+    ;;
+
+  resolve-by-urls)
+    [[ $# -gt 0 ]] || abort "provide at least one discussion_r URL"
+    # Extract numeric ids and reuse resolver
+    # Provide a wrapper that applies NF filter without spawning a subshell that loses function scope
+    extract_ids_filtered() {
+      extract_ids_from_urls "$@" | awk 'NF'
+    }
+    readarray_compat ids extract_ids_filtered "$@"
+    if [[ ${#ids[@]} -eq 0 ]]; then
```

Shellcheck (0.10.0)

[warning] ids is referenced but not assigned.

(SC2154)